### PR TITLE
Start changelog for 3.0 RC3

### PR DESCRIPTION
**Pull request by msullivan**
This has a huge TODO for `pgvector`.

### CODE REVIEW

File: docs/changelog/3_x.rst [hunk 1]
```diff
@@ -927,3 +927,67 @@ Other changes and fixes
...
+
+Support for ``pgvector``
+------------------------
+As a surprise bonus feature, we have introduced support for using the
+`pgvector <pgvector_>`_ Postgres extension for vector similarity search.
+
+This can be used to store and efficiently retrieve text embeddings,
+such as those produced by OpenAI.
+
+The Postgres that comes packaged with the edgedb server includes
```
Comment (raddevon) [inline docs/changelog/3_x.rst:942]:
```suggestion
The Postgres that comes packaged with the EdgeDB server includes
```

File: docs/changelog/3_x.rst [hunk 2]
```diff
@@ -927,3 +927,67 @@ Other changes and fixes
...
+* Fix casts from json to custom scalars
+  (:eql:gh:`#5624`)
+
+* Fix ``pg_namespace`` emulation in SQL layer
+  (:eql:gh:`#5594`)
+
+* Fix json output of arrays of ranges
+  (:eql:gh:`#5629`)
+
+* Fix ISE when using ``_subject__`` in default
```
Comment (raddevon) [inline docs/changelog/3_x.rst:982]:
```suggestion
* Fix ISE when using ``__subject__`` in default
```

File: docs/changelog/3_x.rst [hunk 3]
```diff
@@ -927,3 +927,67 @@ Other changes and fixes
...
+  (:eql:gh:`#5633`)
+
+* Compile default expressions in the schema consistently compile
```
Comment (raddevon) [inline docs/changelog/3_x.rst:985]: I think this one needs a rewrite, but I'm not 100% sure what it says. I'm having trouble parsing it.

**Review by raddevon — APPROVED** [review]
This looks pretty good. I left a couple of suggestions. I'll go ahead and approve in case we get in a bind and need to run with what we have.

**Review by 1st1 — APPROVED** [review]
Looks good, but we'll merge rc3 section into the main changelog sections once 3.0 is out.

**Pull request merged**